pkg/policy/api/rule.go

```diff
@@ -134,6 +144,11 @@ type Rule struct {
 	//
 	// +kubebuilder:validation:Optional
 	Description string `json:"description,omitempty"`
+
+	// Log specifies custom policy-specific Hubble logging configration.
```

Comment:
Spelling nit:

```suggestion
	// Log specifies custom policy-specific Hubble logging configuration.
```

(and in the generated CRD YAML)